sru/release-27/test_xenial_upgrade.sh

```diff
@@ -65,7 +65,8 @@ function teardown_vm() {
 function install_new_ua() {
     tool=$1
     h2 "Set up to use daily ppa"
-    $tool exec $name -- sudo add-apt-repository ppa:ua-client/daily -y
+    $tool exec $name -- sudo sh -c "cat <<EOF >/etc/apt/sources.list.d/ubuntu-$series-proposed.list
```

Comment:
@lucasmoura Really sorry I should've mentioned - this script doesn't currently have `$series` set. Can you set `series=xenial` at the top of the script?